tools/cockpit.spec

```diff
@@ -374,10 +374,10 @@ The Cockpit component for managing storage.  This package uses Storaged.
 
 %package ostree
 Summary: Cockpit user interface for rpm-ostree
-%if 0%{?rhel}
-Requires: rpm-ostree-client >= 2015.11-1
-%else
+%if 0%{?fedora} > 0 && 0%{?fedora} < 24
```

Comment:
There is also some similar confusion on rhel, that's why i thought the more generic option might be ok.

But if you think a targeted centos only change is better we can go with that.